packages/manager/modules/anthos/src/anthos-tenants.service.js

```diff
@@ -31,4 +31,23 @@ export default class AnthosTenantsService {
 
     return guides;
   }
+
+  getHosts(serviceName, additional, pageNumber, pageSize) {
+    return this.$http.get(
+      `/dedicated/anthos/tenants/${serviceName}/baremetals`,
+      {
+        headers: {
+          Pragma: 'no-cache',
```

Comment:
😞

Reply:
Since we have pagination, we would be limiting the number of records fetched... We do not need the cache here, since the status of the hosts keep changing